src/Shells/StrixMusic.Shells.ZuneDesktop/Styles/Items/TrackItemStyle.xaml

```diff
@@ -1,24 +1,50 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:items="using:StrixMusic.Sdk.WinUI.Controls.Items"
+    xmlns:items="using:StrixMusic.Shells.ZuneDesktop.Controls.Views.Items"
     xmlns:viewModels="using:StrixMusic.Sdk.ViewModels"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
     x:Class="StrixMusic.Shells.ZuneDesktop.Styles.Items.TrackItemStyle">
 
     <ResourceDictionary.MergedDictionaries>
         <ResourceDictionary Source="ms-appx:///StrixMusic.Shells.ZuneDesktop/Resources/Brushes.xaml"/>
     </ResourceDictionary.MergedDictionaries>
-    
-    <Style x:Key="ZuneDesktopTrackControlStyle" TargetType="items:TrackItem">
+
+    <Style x:Key="ZuneDesktopTrackControlStyle" TargetType="items:ZuneTrackItem">
         <Style.Setters>
             <Setter Property="Template">
                 <Setter.Value>
-                    <ControlTemplate TargetType="items:TrackItem">
-                        <Grid Background="Transparent" Margin="-24,0" Padding="24,0">
-                            <TextBlock x:Name="PART_Tb" Text="{Binding Track.Name, RelativeSource={RelativeSource Mode=TemplatedParent}}" FontSize="12" FontWeight="SemiBold" Opacity=".8"
+                    <ControlTemplate TargetType="items:ZuneTrackItem">
+                        <Grid Background="Transparent" Margin="-24,0" Padding="24,0" RequestedTheme="Light">
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="2*"/>
+                                <ColumnDefinition Width="1*"/>
+                                <ColumnDefinition Width="1*"/>
+                            </Grid.ColumnDefinitions>
+                            <TextBlock x:Name="PART_Tb" Grid.Column="0" Text="{Binding Track.Name, RelativeSource={RelativeSource Mode=TemplatedParent}}" FontSize="12" FontWeight="SemiBold" Opacity=".8"
                                         VerticalAlignment="Center"/>
-                      
-                        <VisualStateManager.VisualStateGroups>
+
+                            <Rectangle Width="50" HorizontalAlignment="Right" Margin="0,0,0,0" Grid.Column="0">
+                                <Rectangle.Fill>
+                                    <LinearGradientBrush StartPoint="0.5,0" EndPoint="0,0" >
+                                        <GradientStop Offset="0" Color="White" />
+                                        <GradientStop Offset="0.6" Color="Transparent" />
+                                    </LinearGradientBrush>
+                                </Rectangle.Fill>
+                            </Rectangle>
+
+                            <TextBlock Text="{Binding ArtistString, RelativeSource={RelativeSource Mode=TemplatedParent}, Mode=OneWay}" Grid.Column="1" Grid.ColumnSpan="2"/>
+
+                            <Rectangle Width="50" HorizontalAlignment="Right" Margin="0,0,0,0" Grid.Column="1" Grid.ColumnSpan="2">
```

Comment:
This seems to be happening on mouseover and on selection. I'd love to approve this PR, but I don't want us to need to come back to this in the future - one PR and done.

![image](https://user-images.githubusercontent.com/9384894/173247211-c5e79910-9314-4ea5-9a4d-c924589f0086.png)